dlt/common/storages/transactional_file.py

```diff
@@ -114,8 +114,8 @@ def _sync_locks(self) -> t.List[str]:
             output.append(name)
         if not output:
             raise RuntimeError(
-                f"When syncing locks for path {self.path} and lock {self.lock_path} no lock file"
-                " was found"
+                f"Lock syncing failed. No lock file found for path `{self.path}` and lock"
```

Comment:
praise: I like this format much better. is there a wor for it? 
in emails or conversations, I call it `frontloading`